### PR TITLE
Rework Homepage

### PR DESCRIPTION
**Pull request by TomDijkema**
PR reworks Homepage conform the design. It includes the specimen type filters on the left (however, not functional yet) and the search on the right, with the option to toggle the advanced search options.

Adds:
- Specimen Type filters (not functional)  to Home Page
- Advanced filters toggle

Modifies:
- Completely reworks Home Page conform design, content now aligns based on the center which should deal with height responsiveness

Removes:
- Background image from Home Page
- Old introduction text and search components

### CODE REVIEW

File: src/components/home/components/IntroText.tsx [hunk 1]
```diff
@@ -1,28 +1,35 @@
...
-                        Human experts and machines can add annotations on Digital Specimens.
-                        The annotations are stored as FAIR Digital Objects (with a persistent identifier).
-                        UCAS also keeps track of the transactions on the data as provenance information related
-                        to the curation or annotation events.
+                {/* Intro Text */}
+                <Row className="mt-4">
+                    <Col>
+                        <p className={styles.introText}>
+                            Help us with improving the completeness and quality of information about specimens.
+                            We aim to provide all information known about specimens held in Europe to science and to
```
Comment (samleeflang) [inline src/components/home/components/IntroText.tsx:26]: Where did the text come from? This sentence seems a bit difficult.

**Review by samleeflang — APPROVED** [review]
Yes a question about the text on the homepage. Is this something coming from Jose, he might help get the text right.

File: src/components/home/components/IntroText.tsx (hunk 1 above)
Comment (TomDijkema) [reply src/components/home/components/IntroText.tsx:26]: The text comes directly from the design. Looked it up, Wouter wrote this text and provided it to Flores.

Comment (samleeflang) [reply src/components/home/components/IntroText.tsx:26]: Ah perfect then let's merge this one

**Pull request merged**